src/common/admin_socket.cc

```diff
@@ -484,6 +484,27 @@ int AdminSocket::unregister_command(std::string_view command)
   return ret;
 }
 
+void AdminSocket::unregister_commands(const AdminSocketHook *hook)
+{
+  if (hook) {
```

Comment:
under what circumstances, we will have a `nullptr` hook?

Reply:
update.